src/main/java/org/owasp/validator/html/Policy.java

```diff
@@ -24,20 +24,30 @@
 
 package org.owasp.validator.html;
 
-import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import static org.owasp.validator.html.util.XMLUtil.getAttributeValue;
```

Comment:
Can you put these two imports back in their original order? Normally they are in alphabetical order.

Reply:
It's done.